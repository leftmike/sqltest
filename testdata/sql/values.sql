--
-- Test VALUES
--
-- {{if eq Dialect "sqlite3"}}{{Skip}}{{end}}

VALUES (1, 2, 3), (4, NULL, 5);

VALUES (1.2, 3), (4, 5.6);

{{Fail .Test}}
VALUES (1, 2), ('three', 4);

{{Fail .Test}}
VALUES ('one', 2), (3, 4);

{{Fail .Test}}
VALUES (1, 'two'), (3, 4);

{{Fail .Test}}
VALUES (1, 2), (3, 'four');

