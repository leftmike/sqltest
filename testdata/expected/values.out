--
-- Test VALUES
--
-- {{if eq Dialect "sqlite3"}}{{Skip}}{{end}}
VALUES (1, 2, 3), (4, NULL, 5);
   column1 column2 column3
   ------- ------- -------
 1       1       2       3
 2       4               5
(2 rows)
VALUES (1.2, 3), (4, 5.6);
   column1 column2
   ------- -------
 1     1.2       3
 2       4     5.6
(2 rows)
{{Fail .Test}}
VALUES (1, 2), ('three', 4);
{{Fail .Test}}
VALUES ('one', 2), (3, 4);
{{Fail .Test}}
VALUES (1, 'two'), (3, 4);
{{Fail .Test}}
VALUES (1, 2), (3, 'four');
